template/package.json

```diff
@@ -34,7 +34,7 @@
     "css-loader": "^0.28.0",
     "cssnano": "^3.10.0",
     {{#lint}}
-    "eslint": "^3.19.0",
+    "eslint": "^4.4.0",
```

Comment:
Thats right, but since the version string is **^**4.4.0 is should always install the newest available version < 5.0.0.